alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-file-upload-dialog/decision-document-upload-dialog.component.html

```diff
@@ -34,6 +37,9 @@ <h4>{{ title }} Document</h4>
           <mat-icon>close</mat-icon>Remove
         </button>
       </div>
+      <mat-error class="left" style="display: flex" *ngIf="showVirusError">
+        <mat-icon>warning</mat-icon>&nbsp;A virus was detected in the file. Choose another file and try again.
```

Comment:
I forgot which one we agreed on: virus vs malware?

Reply:
This is what is in the Figma.